src/app/public/modules/flyout/flyout.service.spec.ts

```diff
@@ -39,6 +45,7 @@ import {
 describe('Flyout service', () => {
   let service: SkyFlyoutService;
   let applicationRef: ApplicationRef;
+  let router: Router;
 
   beforeEach(() => {
     TestBed.configureTestingModule({
```

Comment:
I seem to remember that it's good practice to import the `RouterTestingModule` into the `TestBed`. 
https://angular.io/api/router/testing/RouterTestingModule

Reply:
Might need it in the flyout component spec, too.

Reply:
It is :) It is imported via the `SkyFlyoutFixturesModule` :)